habitat-lab/habitat/datasets/rearrange/rearrange_dataset.py

```diff
@@ -223,7 +224,15 @@ def from_json(
                 "recep_category_to_recep_category_id"
             ]
 
-        for i, episode in enumerate(deserialized["episodes"]):
+
+        episodes_index_low, episodes_index_high = self.episode_indices_range
+        episode_ids_subset = None
+        all_episodes = deserialized["episodes"]
+        if len(self.config.episode_ids) > 0:
+            episode_ids_subset = self.config.episode_ids[episodes_index_low: episodes_index_high]
```

Comment:
From what I understand, here you are trying to get the list of episode ids for evaluation from the dataset config and then using another config parameter to get a subset of the range from the list?

Reply:
I think combining the two is an overkill. Lets keep the two functionality separate?

Reply:
Yes that is right. If we select some 100 episodes, we can split it into multiple parts again. I was actually using this functionality when I was re-running evals on selected episodes for generating visualizations.